### ISSUE TITLE
Lombok 1.16.8

### ISSUE THREAD

**Issue by mikeholler** [mplushnikov/lombok-intellij-plugin#192]
Lombok 1.16.8 came out recently ([changelog]()), and its new features break this plugin in a couple ways. Below are the changes that I think will have to be added in order to support the new version. As far as I know, none of these enhancements are supported in lombok-intellij-plugin yet.

```
FEATURE: @Builder updates: It now generates clearFieldName() methods if @Singular is used. Issue #967.
FEATURE: @Builder updates: The annotation can now be put on instance methods. Issue #63.
FEATURE: @Builder updates: @Singular now supports guava's ImmutableTable Issue #937.
FEATURE: Putting @Wither on abstract classes now generates something slightly more useful: An abstract wither method. Issue #945.
```

**Comment by AlexejK** [comment]
This does not __break__ the plugin from what I see, however there is that disconnect where the new features are not detected by the plugin. The code that was working before, should work now as well. 

Am I correct in that assumption?

**Comment by mikeholler** [comment]
Sorry @AlexjK. Yes, if you upgrade lombok and do not use any of the new features the plugin works just fine. I meant "break" in the sense that I assumed this plugin's goal was full support of Lombok features, and the gap between the new features and what this plugin supports cases it to (hopefully only temporarily) fall short of this goal. Definitely should have used a better word.

**Comment by mplushnikov** [comment]
The new lombok 1.16.8 features should be available in the next plugin release (version >=0.12)
